Assets/HoloToolkit/Common/Scripts/Singleton.cs

```diff
@@ -6,12 +6,13 @@
 namespace HoloToolkit.Unity
 {
     /// <summary>
-    /// Singleton behaviour class, used for components that should only have one instance.
+    /// Singleton behavior class, used for components that should only have one instance.
```

Comment:
Why rename this?
Unsure but I think the spelling is correct here.

Reply:
with the u has to be a British English version, I think bot hare correct enough, my comment spell checker just flagged it.

Reply:
It should probably have the u, to match the spelling of MonoBehaviour.

Reply:
fair enough